sos/report/plugins/openvswitch.py

```diff
@@ -75,12 +76,19 @@ def setup(self):
             "/run/openvswitch/ovs-monitor-ipsec.pid"
         ])
 
+        self.add_copy_spec([
+            os.path.join('/usr/local/etc/openvswitch', 'conf.db'),
+            os.path.join('/etc/openvswitch', 'conf.db'),
+            os.path.join('/var/lib/openvswitch', 'conf.db'),
```

Comment:
Replace `os.path.join` by `path_join` (cf https://github.com/sosreport/sos/blob/ebe6d4e80e7644a568095e352c8c51072836f822/sos/report/plugins/openvswitch.py#L13 and usage on https://github.com/sosreport/sos/blob/ebe6d4e80e7644a568095e352c8c51072836f822/sos/report/plugins/openvswitch.py#L69) and you can remove the `import os` that LGTM raises an alert about.